org.eclipse.winery.workflowmodeler/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "workflow-modeler",
+  "version": "1.0.0",
+  "description": "A Designer For workflow",
+  "scripts": {
+    "start": "webpack-dev-server --inline --progress",
+    "test": "karma start",
+    "build": "rimraf dist && webpack --config config/webpack.prod.js --progress --profile --bail",
+    "mock": "node mock/server.js",
+    "lint": "ng lint"
+  },
+  "licenses": [
+    {
+      "type": "Apache-2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    {
+      "type": "Eclipse Public License-1.0",
+      "url": "http://www.eclipse.org/legal/epl-v10.html"
+    }
+  ],
+  "dependencies": {
+    "@angular/animations": "4.2.5",
```

Comment:
Could you please update the dependencies to the following versions? The reason is that we already gathered the sources for these versions in order to start the IP Review.

```
"@angular/...": "4.3.0"
```